app/models/budget/investment.rb

```diff
@@ -363,20 +364,17 @@ def self.to_csv(investments, options = {})
                   else
                     I18n.t("admin.budget_investments.index.no_admin_assigned")
                   end
-          vals = if investment.valuators.empty?
-                   I18n.t("admin.budget_investments.index.no_valuators_assigned")
-                 else
-                   investment.valuators.collect(&:description_or_name).join(', ')
-                 end
+          vals = ApplicationController.helpers.assigned_valuators(investment)
```

Comment:
Calling a ApplicationController from a model doesn't feel quite right for me.

Reply:
Maybe the whole `to_csv` method should be extracted to a class that encloses the logic... something like BudgetInvestmentCSVExporter... I don't know ... like a ServiceObject?

Reply:
This is the only way I have found to share the function that returns the names, between the model and the view. in any other case I would have to repeat code.

Maybe I could move the calculation of the contents of the table to a service object and call it both in the view and in the csv.

I'm going to see if it works better

Reply:
Well if the problem is just `assigned_valuation_groups` and `assigned_valuators` there's a really easy fix for it: Move those two functions on to the Budget::Investment model. The `to_csv

Reply:
:sob: I'm dumb...

I'll fix in a moment...